src/client/components/input-field/index.css

```diff
@@ -1,3 +1,10 @@
 .input-field {
   display: inline-block;
 }
+
+.rwd-table .input-field table tr {
```

Comment:
This _is_ a temporary solution which eventually will evolve to a proper solution when new components introduced which are includes `table`s inside their `pug`. But for now, in order to overcome [this issue](https://github.com/aktos-io/scada.js/issues/63#issuecomment-278657296), I suggest that we take advantage of it.